test/parallel/test-worker-init-failure.js

```diff
@@ -35,7 +35,6 @@ if (process.argv[2] === 'child') {
     // (i.e. single cpu) `ulimit` may not lead to such an error.
 
     worker.on('error', (e) => {
-      assert.match(e.message, /EMFILE/);
       assert.ok(e.code === 'ERR_WORKER_INIT_FAILED' || e.code === 'EMFILE');
```

Comment:
I am not sure if this will work: as in the error site in the related failure incident, e.code is going to be `ENOENT`?

Reply:
and worker did not fail in it's `INIT`, instead at runtime, so the assertion at line 38 would fail anyways?

Reply:
I guess putting `Fixes:` in the commit message was a bit hasty. I still think we should probably remove the message sniffing, even if it doesn't resolve the test flakiness.